cmd/crictl/exec.go

```diff
@@ -109,6 +111,20 @@ var runtimeExecCommand = &cli.Command{
 			Aliases: []string{"n"},
 			Usage:   "Exec command for all last n containers, set to 0 for unlimited",
 		},
+		&cli.BoolFlag{
+			Name:    "quiet",
```

Comment:
This is great! Makes it easier to pipe the output to something like `grep`, etc.